students/20th/junmopark/main.html

```diff
@@ -0,0 +1,175 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="common.css">
+    <link rel="stylesheet" href="main.css">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Lobster&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.3/css/all.min.css"
+        integrity="sha512-iBBXm8fW90+nuLcSKlbmrPcLa0OT92xO1BIsZ+ywDWZCvqsWgccV3gFoRBv0z+8dLJgyAHIhR35VZc2oM/gI1w=="
+        crossorigin="anonymous" />
+    <title>Instagram</title>
+</head>
+<body>
+    <section>
+        <nav class="topNav">
+            <header class="navTopItem">
+                <h1 class="title">Instagram</h1>
+            </header>
+            <div class="navTopItem">
+                <input type="text" class="search" placeholder="검색">
+            </div>
+            <div class="navTopItem">
+                <img alt="quest" src="https://s3.ap-northeast-2.amazonaws.com/cdn.wecode.co.kr/bearu/explore.png">
+                <img alt="like" src="https://s3.ap-northeast-2.amazonaws.com/cdn.wecode.co.kr/bearu/heart.png">
+                <img alt="profile" src="https://s3.ap-northeast-2.amazonaws.com/cdn.wecode.co.kr/bearu/profile.png">
+            </div>
+        </nav>
+        <main>
+            <div class="feed">
+                <header>
+                    <div class="headerLeft">
+                        <div>
+                            <imgalt="feedProfileImg" class="ProfileImg" src="/students/20th/junmopark/ Assets/profileImg.jpeg">
```

Comment:
```suggestion
                            <img alt="feedProfileImg" class="ProfileImg" src="/students/20th/junmopark/ Assets/profileImg.jpeg">
```
로 해주셔야 합니다.

<img width="335" alt="스크린샷 2021-04-23 오후 6 40 37" src="https://user-images.githubusercontent.com/54208214/115852759-66e60400-a463-11eb-8de7-d657ad30a505.png">

img 태그가 아닌 위와 같이 이상한 태그로 표시가 되어서 모든 레이아웃이 깨져보이네요
html에서 제공하는 태그가 아닌 이상한 태그명을 사용할 경우에는 브라우저마다 div등의 특정 무의미한 태그로 처리하게 됩니다 태그명 신경써주세요

Reply:
수정했습니다!